target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz5490.dts

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "vr9_avm_fritzxx90.dtsi"
+
+/ {
+	compatible = "avm,fritz5490", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 5490/5491";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+};
+
+&eth0 {
+	mtd-mac-address = <&urlader 0xcc>;
+	mtd-mac-address-increment = <1>;
+};
+
+&gswip_mdio {
+	phy0: ethernet-phy@0 {
+		status = "disabled";
+	};
+
+	phy1: ethernet-phy@1 {
+		status = "disabled";
+	};
+
+	phy5: ethernet-phy@5 {
+		reg = <0x05>;
+	};
+
+	phy6: ethernet-phy@6 {
+		reg = <0x06>;
+		reset-gpios = <&gpio 32 GPIO_ACTIVE_LOW>;
+	};
+
+	phy9: ethernet-phy@9 {
+		reg = <0x09>;
+	};
+
+	phy11: ethernet-phy@11 {
+		status = "disabled";
+	};
+
+	phy13: ethernet-phy@13 {
+		status = "disabled";
+	};
+};
+
+&gswip_ports {
+	port@0 {
+		reg = <0>;
+		label = "wan";
+		phy-mode = "sgmii";
+		phy-handle = <&phy6>;
+	};
+
+	port@1 {
```

Comment:
Is there a Qualcom switch or second Soc connected to this port?

Reply:
The QCA8K switch is connected to port1 (lan3) and the fiber port is phy 6.

Reply:
I would also remove the port connected to the QCA8K switch from the dts if it is not properly supported. It may be added in the future.